dsls/sql/src/main/java/org/apache/beam/dsls/sql/interpreter/operator/arithmetic/BeamSqlArithmeticExpression.java

```diff
@@ -80,12 +80,10 @@ public BeamSqlArithmeticExpression(List<BeamSqlExpression> operands) {
 
   private double getDouble(BeamSqlRow inputRecord, BeamSqlExpression op) {
     Object raw = op.evaluate(inputRecord).getValue();
-    Double ret = null;
     if (SqlTypeName.NUMERIC_TYPES.contains(op.getOutputType())) {
-      ret = ((Number) raw).doubleValue();
+      return ((Number) raw).doubleValue();
     }
-
-    return ret;
+    return 0d;
```

Comment:
Is returning 0 really the right thing to do here? NullPointerException probably isn't ideal, either. How about throwing an IllegalStateException? Or are there cases where we actually expect this to case to happen and also be valid?